### PR TITLE
Include timezone in changelog

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: tasks/changelog-update.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ fi
 /bin/bash tasks/changelog-view.sh 1> new
 
 WORDC=$(wc -w new | tr -d '[:alpha:][:blank:][:punct:]/-')
-DATE=$(date -u +"%Y-%m-%d %H:%M")
+DATE=$(date -u +"%Y-%m-%d %H:%M %Z")
 
 if [ $WORDC -gt 0 ]; then
 	echo "## "$RELEASE" ("$DATE" UTC)" > temp
```
Comment (hakubo) [inline tasks/changelog-update.sh]: so should we remove this hardcoded UTC here?

Comment (rogatty) [inline tasks/changelog-update.sh]: From http://unixhelp.ed.ac.uk/CGI/man-cgi?date:
```
  -u, --utc, --universal
	      print or set Coordinated Universal Time
```
Timezone is always UTC so it doesn't matter if it's hardcoded or added with `%Z`

**Comment by rogatty** [comment]
@gabrys do you agree that this PR is not needed?

**Comment by gabrys** [comment]
I thought I already closed it. Sorry about it

**Pull request closed without merging**